patches/minecraft/net/minecraft/client/particle/ParticleManager.java.patch

```diff
@@ -1,6 +1,14 @@
 --- a/net/minecraft/client/particle/ParticleManager.java
 +++ b/net/minecraft/client/particle/ParticleManager.java
-@@ -63,7 +63,7 @@
+@@ -28,6 +28,7 @@
+ import net.minecraft.client.renderer.ActiveRenderInfo;
+ import net.minecraft.client.renderer.BufferBuilder;
+ import net.minecraft.client.renderer.Tessellator;
++import net.minecraft.client.renderer.culling.ICamera;
```

Comment:
Import in patch, please remove

Reply:
Uhhh, I don't quite understand, the patch requires this import to work?

Reply:
No it doesn't - references to "ICamera" can be replaced with "net.minecraft.client.renderer.culling.ICamera" without the import.

Reply:
Yeah, just as I sent that I realised my mistake, generating new patches now :P